src/core_plugins/kibana/common/field_formats/types/_numeral.js

```diff
@@ -1,6 +1,6 @@
 import _ from 'lodash';
-import numeral from 'numeral';
-import { FieldFormat } from 'ui/index_patterns/_field_format/field_format';
+import numeral from '@spalger/numeral';
```

Comment:
Should we consider moving this to `elastic/numeral` now that https://github.com/elastic/numeral-js exists?